MAINTAINERS.md

```diff
@@ -4,7 +4,7 @@
 
 | Maintainer       | GitHub ID                                              | Affiliation                              |
 |------------------|--------------------------------------------------------|------------------------------------------|
-| Anusha Hegde     | [anusha94](https://github.com/anusha94 )               | [VMware](https://www.github.com/vmware/) |
```

Comment:
Time to create a new list for Emeritus maintainers :)
Refer https://github.com/vmware-tanzu/velero/blob/main/MAINTAINERS.md#emeritus-maintainers

Reply:
Thanks for suggesting this.